src/_blog/webtransport.md

```diff
@@ -0,0 +1,247 @@
+---
+tags:
+- browser, transport
+title: WebTransport in libp2p
```

Comment:
Being part of the Universal Browser Connectivity series, it is probably worth having a subheading: 
```
WebTransport in libp2p
Part 1 of Universal Browser Connectivity
```